### PR TITLE
Updating rules to negate proofpoint & cisco encrypted shares

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: detection-rules/attachment_any_html_in_archive_unsolicited.yml [hunk 1]
```diff
@@ -20,6 +20,15 @@ source: |
       and not profile.by_sender().any_false_positives
     )
   )
+
+  // Negating IPHMX.com used in a secure message delivery capacity
```
Comment (jkamdjou) [inline detection-rules/attachment_any_html_in_archive_unsolicited.yml:24]:
```suggestion
  // Negating IPHMX.com (Cisco) used in a secure message delivery capacity 
```

File: detection-rules/attachment_any_html_new_sender.yml [hunk 2]
```diff
@@ -33,6 +33,14 @@ source: |
     or sender.email.domain.root_domain not in $high_trust_sender_root_domains
   )
 
+  // Negating IPHMX.com used in a secure message delivery capacity
```
Comment (jkamdjou) [inline detection-rules/attachment_any_html_new_sender.yml:36]:
```suggestion
  // Negating IPHMX.com (Cisco) used in a secure message delivery capacity 
```

File: detection-rules/attachment_any_html_unsolicited.yml [hunk 3]
```diff
@@ -29,6 +29,15 @@ source: |
     )
     or sender.email.domain.root_domain not in $high_trust_sender_root_domains
   )
+  
+  // Negating IPHMX.com used in a secure message delivery capacity
```
Comment (jkamdjou) [inline detection-rules/attachment_any_html_unsolicited.yml:33]:
```suggestion
  // Negating IPHMX.com (Cisco) used in a secure message delivery capacity 
```

File: detection-rules/link_credential_phishing_secure_message.yml [hunk 4]
```diff
@@ -32,7 +32,9 @@ source: |
                         strings.contains(.value, 'multipart/mixed; boundary="PROOFPOINT_BOUNDARY_1"')
             )
     )
+    and not strings.ends_with(headers.message_id, ".iphmx.com>")
```
Comment (jkamdjou) [inline detection-rules/link_credential_phishing_secure_message.yml:35]:
```suggestion
    // Negating IPHMX.com (Cisco) used in a secure message delivery capacity 
    and not strings.ends_with(headers.message_id, ".iphmx.com>")
```

**Review by jkamdjou — APPROVED** [review]

File: detection-rules/attachment_any_html_new_sender.yml [hunk 5]
```diff
@@ -10,7 +10,14 @@ type: "rule"
 severity: "medium"
 source: |
   type.inbound
-  and any(attachments, .file_extension in~ ('htm', 'html') or .file_type == "html")
+  and any(attachments,
+          (.file_extension in~ ('htm', 'html') or .file_type == "html")
+          
+          // negate cisco secure spawned HTML
+          and not regex.match(file.parse_html(.).raw,
```
Comment (jkamdjou) [inline detection-rules/attachment_any_html_new_sender.yml:17]: a little hesitant to negate on this, feels a little too easy to bypass

Comment (jkamdjou) [reply detection-rules/attachment_any_html_new_sender.yml:17]: thoughts on going back to the iphmx in headers.domains approach? or combined w/ message_id?

Comment (morriscode) [reply detection-rules/attachment_any_html_new_sender.yml:17]: Message ID and iphmx neither are not consistent in the group of FP's :(

Comment (jkamdjou) [reply detection-rules/attachment_any_html_new_sender.yml:17]: how many / what % don't match either of those patterns?

**Pull request closed without merging**